The default font used by VLC does not exists on every system, instead of that,
we use DejaVU, and add it to the dependencies, as anyway many people have that
font installed.

Index: vlc-clean/modules/misc/freetype.c
===================================================================
--- vlc-clean.orig/modules/misc/freetype.c
+++ vlc-clean/modules/misc/freetype.c
@@ -63,8 +63,8 @@
 #define DEFAULT_FONT "" /* Default font found at run-time */
 #define FC_DEFAULT_FONT "Arial"
 #else
-#define DEFAULT_FONT "/usr/share/fonts/truetype/freefont/FreeSerifBold.ttf"
-#define FC_DEFAULT_FONT "Serif Bold"
+#define DEFAULT_FONT "/usr/share/fonts/dejavu/DejaVuSans.ttf"
+#define FC_DEFAULT_FONT "Sans"
 #endif
 
 #if defined(HAVE_FRIBIDI)
